.github/workflows/linux.yml

```diff
@@ -65,23 +65,9 @@ jobs:
 
     name: linux debug parallel simplex
     runs-on: [ubuntu-22.04]
+    container:
+      image: dealii/dealii:master-jammy
```

Comment:
Shouldn't this be `dealii/dependencies:jammy`?

Reply:
Fixed, thanks!